frontend/package.json

```diff
@@ -78,6 +79,7 @@
     "semver": "7.3.5",
     "source-map-explorer": "^2.5.2",
     "style-loader": "^2.0.0",
+    "svg-url-loader": "^7.1.1",
```

Comment:
Is this needed for react-dsfr?

Reply:
yes, it was a missing dependency from our webpack config (see https://github.com/betagouv/sante-psy/blob/b553b99bb7eef2b5ea6cf7ceac96e2377cff0837/frontend/config/webpack.base.js#L53) that is now trigger by the react-dsfr